### PR TITLE
Upadate colors user detail

### PR DESCRIPTION
**Pull request by jon-stevens**
Use the colors from the springernature context, update readme to add missing dep

pr-mobile

### CODE REVIEW

File: toolkits/springernature/packages/springernature-user-details/README.md [hunk 1]
```diff
@@ -52,5 +52,6 @@ Import the enhanced settings and styles into your main stylesheet
 // enhanced.scss
 @import '@springernature/brand-context/default/scss/enhanced';
 @import '@springernature/user-details/scss/10-settings/colours';
+@import '@springernature/user-details/scss/10-settings/layout';
```
Comment (alexkilgour) [inline toolkits/springernature/packages/springernature-user-details/README.md:55]: I think this file is now out of date. @anibe posted a PR this morning that updates this file and bumps the version - https://github.com/springernature/frontend-toolkits/pull/397/files

Comment (anibe) [reply toolkits/springernature/packages/springernature-user-details/README.md:55]: @jon-stevens I updated the links this morning. I think you just need to add the `springer-nature-` to the file name in the docs

Comment (anibe) [reply toolkits/springernature/packages/springernature-user-details/README.md:55]: Ughh it's my fault, I thought I had merged.. If you rebase with master you should get the latest. Apologies

File: toolkits/springernature/packages/springernature-user-details/HISTORY.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # History
 
+## 1.3.2 (2020-11-26)
+	* Use global colours
+
```
Comment (anibe) [inline toolkits/springernature/packages/springernature-user-details/HISTORY.md:5]: Missing the 1.3.1 changelog

Comment (anibe) [reply toolkits/springernature/packages/springernature-user-details/HISTORY.md:5]:
```
## 1.3.1 (2020-11-26)
	* Adds correct filenames to README.md
```

**Review by anibe — CHANGES_REQUESTED** [review]

**Review by anibe — APPROVED** [review]

**Pull request merged**